app/code/community/Ebizmarts/MailChimp/Model/Api/Products.php

```diff
@@ -324,6 +388,7 @@ protected function _buildProductData($product, $magentoStoreId, $isVariant = tru
         //data applied for both root and varient products
         $data["id"] = $productId;
         $data["title"] = $rc->getAttributeRawValue($productId, 'name', $magentoStoreId);
+        if($break)  $data["title"] = $rc->getAttributeRawValue($productId, 'name', $magentoStoreId) ."\xB1\x31";
```

Comment:
Please avoid this one line if statements. Standards recommend always using standard format with brackets.